### ISSUE TITLE
Relative/absolute import in train.py

### ISSUE THREAD

**Issue by Cibale** [fizyr/keras-retinanet#195]
I am not sure why, but these imports doesn't work for me.

Locally, when I run `retinanet-train` command, everything works smoothly. However, if I run (from the same directory) `python keras-retinanet/bin/train.py` I encounter this error:

```
Traceback (most recent call last):
  File "keras_retinanet/bin/train.py", line 34, in <module>
    from .. import losses
```
Now, since it doesn't work locally that way, I didn't even try it in a cloud.

However, when in a cloud I type `retinanet-train` command, it doesn't exist (I guess this is because of `setup.py`)

How can I fix these imports so it works?

**Comment by de-vri-es** [comment]
That is strange, I did test it. Could you try running it again with:
```
PYTHONPATH='.' python keras-retinanet/bin/train.py
```

That shouldn't be necessary, but who knows...

Could you also tell us which python version you're using? (`python --version`)

**Comment by Cibale** [comment]
I tried it, but the output (error) is the same:

```
Using TensorFlow backend.
Traceback (most recent call last):
  File "keras_retinanet/bin/train.py", line 34, in <module>
    from .. import losses
SystemError: Parent module 'keras_retinanet.bin' not loaded, cannot perform relative import

```

Result of `python --version` is `Python 3.5.2`

**Comment by de-vri-es** [comment]
Hmm, I tested with python 3.6. Would it be possible for you to test if python 3.6 works in your scenario?

If it does, we can see if there is a way to achieve python 3.5 compatibility too. If it doesn't, we'll have to dig deeper.

**Comment by Cibale** [comment]
Well, it works with 3.6 :+1: 

Should we close the issue?

**Comment by de-vri-es** [comment]
Very strange indeed. I pretty much followed [PEP-366](https://www.python.org/dev/peps/pep-0366/) to the letter. I'll do some more digging to see if it can be made to work with older python versions.

**Comment by de-vri-es** [comment]
I believe 79a442e62e7f362def00689545682a2fecfe5175 fixes the issue. Python's import system never ceases to amaze me.. The parent module has to be imported before relative imports are possible, even though PEP-366 mentions no such restriction. But apparantly it was fixed in python 3.6.4.

@Cibale: can you test if the issue is solved with 79a442e62e7f362def00689545682a2fecfe5175?

**Comment by de-vri-es** [comment]
I tested it myself with python 2.7.14. The same problem was present before the fix, but not after. I'm assuming it's also fixed for python < 3.6.3. If the issue isn't fix, please let us know here :)

Thanks for reporting this!
